Source/FieldSolver/FiniteDifferenceSolver/EvolveE.cpp

```diff
@@ -144,7 +158,7 @@ void FiniteDifferenceSolver::EvolveECartesian (
             [=] AMREX_GPU_DEVICE (int i, int j, int k){
 #ifdef AMREX_USE_EB
                 // Skip field push if this cell is fully covered by embedded boundaries
-                if (lx(i, j, k) <= 0) return;
+                if (lx(i, j, k) <= 0 or isnan(lx(i,j,k))) return;
```

Comment:
```suggestion
                if (lx(i, j, k) <= 0 or amrex::isnan(lx(i,j,k))) return;
```
and in other places for portability.

Reply:
In general, these isnan's were obsolete so I just removed them. Thanks for the suggestion.